src/main/java/db.properties

```diff
@@ -0,0 +1,4 @@
+dbDriver=com.mysql.jdbc.Driver
```

Comment:
This file should moved to `src/main/resources` source folder (please do this via Eclipse, would be easier).